vignettes/tune.Rmd

```diff
@@ -82,10 +82,10 @@ ames_rec <-
   step_ns(Latitude,  deg_free = tune("lat df"))
 ```
 
-The function `dials::parameters()` can detect and collect the parameters that have been flagged for tuning:
+The function `hardhat::extract_parameter_set_dials()` can detect and collect the parameters that have been flagged for tuning. *This method replaces the function `dials::parameters()` deprecated in tune 0.1.6.9003
```

Comment:
Very nice! it appears that you forgot to add the matching ending `*`. Also finishing the line with a period would be appreciated.

Reply:
Woops, may have stepped on your toes here. Will mark as duplicate.